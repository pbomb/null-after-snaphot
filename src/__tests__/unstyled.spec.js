import React from 'react';
import { render } from 'react-testing-library';
import Unstyled from '../unstyled';

test('container is not null after snapshot', () => {
  const { container } = render(<Unstyled text="test text" />);

  expect(container.firstChild).toMatchSnapshot();
  expect(container.firstChild).not.toBeNull();
});
