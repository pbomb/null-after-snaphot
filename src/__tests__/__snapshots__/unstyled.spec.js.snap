// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container is not null after snapshot 1`] = `
<div>
  test text
</div>
`;
